medical-portal/src/API/RSBC.DMF.MedicalPortal.API/Controllers/ChefsController.cs

```diff
@@ -0,0 +1,177 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Rsbc.Dmf.CaseManagement.Service;
+using RSBC.DMF.MedicalPortal.API.Services;
+using System;
+using System.Collections.Generic;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Pssg.Rsbc.Dmf.DocumentTriage;
+using UploadFileRequest = Pssg.DocumentStorageAdapter.UploadFileRequest;
+using System.Security.AccessControl;
+using RSBC.DMF.MedicalPortal.API.ViewModels;
+
+namespace RSBC.DMF.MedicalPortal.API.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class ChefsController : ControllerBase
+    {
+        private readonly ICaseQueryService caseQueryService;
+        private readonly CaseManager.CaseManagerClient _cmsAdapterClient;
+        // private readonly ILogger<ChefsController> logger;
+
+        public ChefsController(ICaseQueryService caseQueryService, CaseManager.CaseManagerClient cmsAdapterClient)
+        {
+            this.caseQueryService = caseQueryService;
+            _cmsAdapterClient = cmsAdapterClient;
```

Comment:
regarding the Serilog comment below, you can use _logger = loggerFactory.Create<ChefsController>(); here to create the logger